### PR TITLE
🌐 [i18n-KO] Update README_ko.md

### PR DESCRIPTION
**Pull request by J4BEZ**
Delete the blank paragraph in the language selection button and Edit to synchronize with the English version of README.md

# What does this PR do?

Delete an empty paragrah under the language selection by removing white space.
And synchronize `README_ko.md` to [README.md](https://github.com/huggingface/transformers/blob/main/README.md).

- [x] Additional edit for keep consistency with main [documentation](https://huggingface.co/docs/transformers/v4.44.2/ko/index). 
(메인 문서와 일관성 유지를 위한 수정)
- [x] Grammar Check (맞춤법 검사)

* Maintain the content of the existing document as much as possible.
* Juxtapose in English of some tasks.

There could be mistranslations or awkwardness.
If you found any of them, I would appreciate it if you could correct the error to make the community shine🙇‍♂️

Thanks for your hard effort that everyone can enjoy thousands of models and learn DL more easily!

I'm sorry to find this PR late #20179
(굉장히 좋은 가이드가 있었군요! 늦게 확인해서 죄송합니다😅)

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: i18n/README_ko.md [hunk 1]
```diff
@@ -96,24 +137,46 @@ limitations under the License.
...
 
-# Allocate a pipeline for question-answering
->>> question_answerer = pipeline('question-answering')
->>> question_answerer({
-...     'question': 'What is the name of the repository ?',
-...     'context': 'Pipeline has been included in the huggingface/transformers repository'
-... })
-{'score': 0.30970096588134766, 'start': 34, 'end': 58, 'answer': 'huggingface/transformers'}
-
+# Download an image with cute cats
```
Comment (heuristicwave) [inline i18n/README_ko.md:149]:
```suggestion
# 귀여운 고양이가 있는 이미지를 다운로드하세요
```

File: i18n/README_ko.md [hunk 2]
```diff
@@ -96,24 +137,46 @@ limitations under the License.
...
+>>> url = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/coco_sample.png"
+>>> image_data = requests.get(url, stream=True).raw
+>>> image = Image.open(image_data)
+
+# Allocate a pipeline for object detection
```
Comment (heuristicwave) [inline i18n/README_ko.md:154]:
```suggestion
# 객체 감지를 위한 파이프라인을 할당하세요
```

**Review by heuristicwave — COMMENTED** [review]
안녕하세요 @J4BEZ 님, 소중한 변역 감사합니다!

이전에 코드의 주석도 번역하라는 가이드가 있어 변경해보았습니다!

edit_README_ko 에서 README.md 파일도 수정한 것은 Update README_ko.md 과는 분리하는게 좋아보입니다!
@junejae @jungnerd @harheem @nuatmochoi 의견 부탁드려요~

**Comment by J4BEZ** [comment]
@heuristicwave 섬세한 피드백 너무나 감사드립니다🙇‍♂️!!
리뷰하여 주신대로 코드에 나타난 주석을 번역해주신 제안으로 수정하였으며
한국어 README.md에만 집중될 수 있도록 영문 README.md 파일의 링크는 원본으로 되돌렸습니다😆😆

이번 한 주 고생 많으셨습니다🙇‍♂️
토요일 밤, 하루 잘 마무리 하시고 복되고 평안한 주말 되시길 바랍니다🤗

**Comment by Rocketknight1** [comment]
cc @stevhliu for vis

**Review by stevhliu — APPROVED** [review]
LGTM, thanks for updating! Feel free to mark as ready for review when you're done 🤗

**Comment by J4BEZ** [comment]
@stevhliu Thank you for your kind reply😄
I'll change this PR from `Draft` to `Ready for review` soon, after adjusting some improvements!

Thank you for your hard effort and hope you have a great day😄

**Comment by J4BEZ** [comment]
@stevhliu Thank you for waiting😆
I finished translation and synchronization🙌🙌

Hope you have a peaceful day and stay healthy!

**Pull request merged**